week3/homework/src/functions.js

```diff
@@ -0,0 +1,69 @@
+const fs = require('fs');
+
+function add(data, item) {
+  const lastObject = data[data.length - 1];
+  let counter = 0;
+  if (lastObject === undefined) {
+    counter = 0;
+  }
+  else {
+    counter = parseInt(lastObject.id) + 1;
```

Comment:
If you remove your `index` from the JSON data, and just use te index of an array, you don't need a counter; you could just use `data[0]` to get the first and `data.length` to know at which position you need to add the new item. Now when you remove an item, the index isn't correct anymore. 
Try to fix this, and if you don't understand please let me know, because I will explain it to you: it is important to understand this since it is the basics hew to work with objects and arrays, you will use this a lot in a future job!

Reply:
Dont really understand